spec/api/keys/keys_spec.rb

```diff
@@ -876,15 +944,28 @@ def make_user_payload(payload)
           delete("#{platform.server}/users/#{org_user_name}", superuser).should look_like({:status => 200})
         end
 
-        it "by an invalid user fails with a 401", :authentication do
-          get("#{platform.server}/users/#{user['name']}/keys", requestor("bob", user['private_key'])).should look_like({:status => 401})
+        context "when GET /users/user/keys is called (list keys)" do
+          it "by an invalid user fails with a 401", :authentication do
+            get("#{platform.server}/users/#{user['name']}/keys", requestor("bob", user['private_key'])).should look_like({:status => 401})
+          end
+          it "by a client in the same org fails with a 401", :authentication do
+            list_user_keys(org_user_name, org_client).should look_like({:status => 401})
+          end
+          it "who isn't valid by a user who is valid fails with a 404" do
+            get("#{platform.server}/users/bob/keys", org_user).should look_like({:status => 404})
+          end
         end
 
-        it "by a client in the same org fails with a 401", :authentication do
-          list_user_keys(org_user_name, org_client).should look_like({:status => 401})
-        end
-        it "who isn't valid by a user who is valid fails with a 404" do
-          get("#{platform.server}/users/bob/keys", org_user).should look_like({:status => 404})
+        context "when GET /users/user/keys/default is called" do
```

Comment:
would suggest GET /users/:user/keys/:name since the specific value of 'default' isn't important to what we're testing.